smdebug/tensorflow/keras.py

```diff
@@ -739,6 +730,23 @@ def run(*args, **kwargs):
 
         return run
 
+    def _prepare_opt_variables(self):
+        optimizer_variables_collection = self.get_collection(CollectionKeys.OPTIMIZER_VARIABLES)
+        custom_collections, _ = self._get_custom_and_default_collections()
+
+        for tensor_ref in optimizer_variables_collection.get_tensors():
```

Comment:
do we need tensor_ref or is opt variable available as an EagerTensor when gradtape is used?

Reply:
I have removed the `prepare_opt_var` fn